homeassistant/components/lock/nuki.py

```diff
@@ -58,12 +117,21 @@ def is_locked(self):
         """Return true if lock is locked."""
         return self._locked
 
+    @property
+    def device_state_attributes(self):
+        """Return the device specific state attributes."""
+        data = {
+            ATTR_BATTERY_CRITICAL: self._battery_critical,
+            ATTR_NUKI_ID: self._nuki_lock.nuki_id}
+        return data
+
     @Throttle(MIN_TIME_BETWEEN_SCANS, MIN_TIME_BETWEEN_FORCED_SCANS)
```

Comment:
This is old code, but `Throttle` should only be used in a common data instance method to protect from multiple updates from different entity against a common API. Applying the throttle here will just limit updates for the same `NukiLock` entity. It will not limit updates from other `NukiLock` entities during the same time. So for this use case, there's already built in functionality in the `EntityComponent` class that the lock base component uses. Just define `SCAN_INTERVAL` in this module if you want to override the default `SCAN_INTERVAL = timedelta(seconds=30)`.